gradle/base-information.gradle

```diff
@@ -7,8 +7,20 @@
 
 apply plugin: 'base'
 
+File versionFile = file( "${rootProject.projectDir}/gradle/version.properties" )
+
+if ( project.hasProperty( 'releaseVersion' ) ) {
+	if ( !project.hasProperty( 'developmentVersion' ) ) {
+		throw new GradleException(
+				"When releaseVersion parameter is specified also a developmentVersion have to be provided "
```

Comment:
I'd rephrase the error to english:
"When the `releaseVersion` parameter is specified, the parameter `developmentVersion` is required as well."